src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -485,6 +508,24 @@ export default {
 				console.debug(exception)
 			}
 			this.loading = false
+			setTimeout(() => this.getBridgeProcessState(this.token), 4000)
```

Comment:
Why the delay? with the await above it should be good already? Or is this the "startup" time of matterbridge binary? Seems a bit weird to wait a random amount of time?

Reply:
It's weird indeed. I just want to wait that the log file is a little bit populated. But as we get the log file content each time the user wants to see it, the delay now seems useless.

Still, it might not be so useless to wait a few seconds. We might see the process alive right after launching it but it will die in a few seconds. I think it has happened to me a few times.

What do you think?

Reply:
Well in this case I would say `editBridge` should return the info, and in case it was not there (yet) it can be refetched/checked after some time. But randomly refreshing after 4 seconds feels weird, in case it terminates after 30 sec it still shows as functional?